lib/agora/active_stream.dart

```diff
@@ -1,18 +1,29 @@
 import 'dart:async';
-import 'dart:math';
+import 'dart:math' as math;
```

Comment:
The Flutter source code has a convention of always naming the "math" import, so I thought it'd be good to do that in our repo as well.